apps/condo/domains/marketplace/gql.js

```diff
@@ -31,7 +31,7 @@ const MarketItemFile = generateGqlQueries('MarketItemFile', MARKET_ITEM_FILE_FIE
 const MARKET_ITEM_PRICE_FIELDS = `{ price { type group name price isMin vatPercent salesTaxPercent currencyCode } marketItem { id } ${COMMON_FIELDS} }`
 const MarketItemPrice = generateGqlQueries('MarketItemPrice', MARKET_ITEM_PRICE_FIELDS)
 
-const MARKET_PRICE_SCOPE_FIELDS = `{ marketItemPrice { id marketItem { id name sku organization { id } marketCategory { id name parentCategory { id name } } } price { type group name price isMin vatPercent salesTaxPercent currencyCode } } property { id } ${COMMON_FIELDS} }`
+const MARKET_PRICE_SCOPE_FIELDS = `{ marketItemPrice { id marketItem { id name sku organization { id } marketCategory { id name parentCategory { id name } } } price { type group name price isMin vatPercent salesTaxPercent currencyCode } } property { id address addressKey addressMeta { ${ADDRESS_META_SUBFIELDS_QUERY_LIST} } map { ${PROPERTY_MAP_JSON_FIELDS} } } ${COMMON_FIELDS} }`
```

Comment:
do you really need addressMeta fields to receive from api?

Reply:
yes, cause we need to show only street part of address ([layouts](https://www.figma.com/file/uyYAPR012LK5vFO2L6BFn9/%E2%80%94-%D0%9D%D0%BE%D0%B2%D1%8B%D0%B5-%D1%84%D0%B8%D1%87%D0%B8?node-id=14332%3A37631&mode=dev))